model/execution/StateService.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace oat\taoProctoring\model\execution;
+
+use oat\taoDelivery\model\execution\StateService as DeliverStateService;
+
+class StateService extends DeliverStateService
+{
+    public function getDeliveryStates()
```

Comment:
isn't it easier to add it to the ```oat\taoProctoring\model\implementation\DeliveryExecutionStateService``` class ?
Here you will loose every proctoring related behaviour